### ISSUE TITLE
return correct comm size

### ISSUE THREAD

**Issue by jacobmerson** [SCOREC/redev#33]
I'm not sure if the following behavior is intended or not. If `GetClientCommSize` is called on the client then it always returns a size of zero. Also, if `GetServerCommSize` is called on the server it always returns zero.

It seems like these functions should return the correct size both on the client and server. Alternatively, this could be split into two functions e.g., `SendClientCommSize` which is only available on the RDV client and `ReceiveClientCommSize` which is only available on the server.

Another alternative might be to rename `GetServerCommSize`  to `SendReceiveServerCommSize`? This seems more clear to me that this is about doing the communication of communicator size.

https://github.com/SCOREC/redev/blob/9a4432b510bb5f01b3ba6010e741c4068977ddd9/redev.cpp#L358-L409

**Comment by cwsmith** [comment]
Yeah... definitely not the intended use ... just sloppy programming/function naming.

`SendReceiveServerCommSize` and `SendReceiveClientCommSize` are much better names.
